cli/tsc/00_typescript.js

```diff
@@ -56362,7 +56362,7 @@ function createTypeChecker(host) {
         if (
           // Check for unusable parameters symbols
           symAtLocation === unknownSymbol || // If the symbol is not found, but was not found in the original scope either we probably have an error, don't reuse the node
-          symAtLocation === void 0 && sym !== void 0 || // If the symbol is found both in declaration scope and in current scope then it shoudl point to the same reference
+          symAtLocation === void 0 && sym !== void 0 || // If the symbol is found both in declaration scope and in current scope then it should point to the same reference
```

Comment:
Can you revert these ones? It's in generated code.

Reply:
Reverted my change as requested.